lib/actions/scrollTo.js

```diff
@@ -4,10 +4,10 @@ const { defaultConfig, setNavigationOptions } = require('../config');
 const { highlightElement } = require('../elements/elementHelper');
 const { doActionAwaitingNavigation } = require('../doActionAwaitingNavigation');
 
-const scrollToElement = async (element) => {
+const scrollToElement = async (element, alignments) => {
```

Comment:
I think it's better to call this alignment

```suggestion
const scrollToElement = async (element, alignment) => {
```